blockchain/fullblocktests/generate.go

```diff
@@ -756,6 +756,16 @@ func (g *testGenerator) assertTipBlockMerkleRoot(expected chainhash.Hash) {
 	}
 }
 
+// BlockByName returns the block associated with the provided block name.  It
```

Comment:
`BlockByName` should be lowercase since the function isn't exported

Reply:
Okay, made the change, check please